runtime-common/stdlib/serialization/serialization-context.h

```diff
@@ -13,6 +16,16 @@ struct SerializationLibContext final : private vk::not_copyable {
   // msgpack serialization
   size_t instance_depth{0};
   uint32_t serialize_as_float32_{0};
+  std::optional<std::array<char, 256>> msgpack_error{std::nullopt};
```

Comment:
Just comment - default constructor of std::optional will create an optional without value